quiz.js

```diff
@@ -0,0 +1,66 @@
+var arr=[];
+var index = 0;
+//store user input controls & submit button
+var submitBtn = document.getElementById("submit");
+var height = document.getElementById("height");
+var character = document.getElementById("character");
+
+
+
+function growTree(height, character){
+
```

Comment:
Doublecheck the requirements. They state that this functions needs to take an object as its single argument. So, in your click event, build an object out of the height and character and pass it in on the function call.